content/en/synthetics/api_tests/grpc_tests.md

```diff
@@ -0,0 +1,186 @@
+
+3. **Name** your gRPC test.
+
+4. Add `env` **Tags** as well as any other tag to your gRPC test. You can then use these tags to quickly filter through your Synthetic tests on the [Synthetic Monitoring homepage][5].
+
+{{< img src="synthetics/api_tests/http_test_config.png" alt="Define HTTP request" style="width:90%;" >}} 
+<!-- TODO replace with gRPC img -->
+
+Click **Test URL** to try out the request configuration. A response preview is displayed on the right side of your screen.
+
+### Define assertions
+
+Assertions define what an expected test result is. When hitting `Test URL` an assertion on the `response time` is added based on the response that was obtained. You must define at least one assertion for your test to monitor.
+
+| Type                    | Operator                                        | Value type                           |
+|-------------------------|-------------------------------------------------|--------------------------------------|
+| response time           | `is less than`                                  | _Integer (ms)_                       |
+| healthcheck status      | `is`, `is not`                                  | _Integer (ms)_                       |
+
+<!-- TODO verify if we can do regex on healthcheck status -->
+You can create up to 20 assertions per API test by clicking on **New Assertion** or by clicking directly on the response preview:
+
+{{< img src="synthetics/api_tests/assertions.png" alt="Define assertions for your HTTP test" style="width:90%;" >}}
+<!-- TODO update screenshot -->
+### Select locations
+
+Select the **Locations** to run your gRPC test from. gRPC tests can run from both [managed][2] and [private locations][3] depending on your preference for running the test from outside or inside your network.
+
+### Specify test frequency
+
+gRPC tests can run:
+
+* **On a schedule** to ensure your most important endpoints are always accessible to your users. Select the frequency at which you want Datadog to run your gRPC test.
+* [**Within your CI/CD pipelines**][4] to start shipping without fearing faulty code might impact your customers experience.
+* **On-demand** to run your tests whenever makes the most sense for your team.
+
+### Define alert conditions
+
+Set alert conditions to determine the circumstances under which you want a test to fail and trigger an alert.
+
+#### Alerting rule
+
+When you set the alert conditions to: `An alert is triggered if any assertion fails for X minutes from any n of N locations`, an alert is triggered only if these two conditions are true:
+
+* At least one location was in failure (at least one assertion failed) during the last *X* minutes;
+* At one moment during the last *X* minutes, at least *n* locations were in failure.
+
+#### Fast retry
+
+Your test can trigger retries `X` times after `Y` ms in case of a failed test result. Customize the retry interval to suit your alerting sensibility.
```

Comment:
```suggestion
Your health check test can trigger retries `X` times after `Y` ms in case of a failed test result. Customize the retry interval to suit your alerting sensibility.
```